pkg/reconciler/build/validation_test.go

```diff
@@ -43,7 +43,7 @@ func TestValidateBuild(t *testing.T) {
 				Template: &v1alpha1.TemplateInstantiationSpec{
 					Arguments: []v1alpha1.ArgumentSpec{{
 						Name:  "foo",
-						Value: "hello",
+						Value: "hello-world",
```

Comment:
These changes don't seem necessary.

Reply:
This is a WIP but these changes are unfortunately necessary for (until I change some pieces around) as these tests hit MakePod and MakePod now calls out to the internet to fetch image metadata.  This is a temporary hack as "hello-world" is a docker image but this is obvi not great as now unit tests have an external image dependence and require internet access.  I am working on fixing this.